lib/types/btrfs.nix

```diff
@@ -75,12 +133,14 @@
       inherit config options;
       default = ''
         mkfs.btrfs ${config.device} ${toString config.extraArgs}
+        ${partitionSwapCreate config.device config.swap}
         ${lib.concatMapStrings (subvol: ''
           (
             MNTPOINT=$(mktemp -d)
             mount ${config.device} "$MNTPOINT" -o subvol=/
             trap 'umount $MNTPOINT; rm -rf $MNTPOINT' EXIT
             btrfs subvolume create "$MNTPOINT"/${subvol.name} ${toString subvol.extraArgs}
+            ${subvolSwapCreate ''"$MNTPOINT"/${subvol.name}'' subvol.swap}
```

Comment:
```suggestion
            ${lib.optionalString subvol.swap.enable ''
              btrfs filesystem mkswapfile --size ${subvol.swap.size} "$MNTPOINT"/${subvol.swap.path}
            ''}
```

Reply:
I guess it would be easier to inline this? would make the code more compact

Reply:
I personally don't like multiline and re-entrant string-interpolations, but that's just my personal preference. I can definitely make this change if that's what preferred.

Reply:
Do you still feel it's better to inline after I've added support for multiple swapfiles?